components/brave_rewards/browser/rewards_service.h

```diff
@@ -310,6 +310,8 @@ class RewardsService : public KeyedService {
 
   virtual void GetExternalWallet(GetExternalWalletCallback callback) = 0;
 
+  virtual std::vector<std::string> GetExternalWalletProviders() = 0;
```

Comment:
`const std::vector<std::string>` and also this method should be `const`

Reply:
fixed https://github.com/brave/brave-core/pull/8566/files#diff-8db01f5ca735055c54fa750b7ac6c4a99e35dffb769d59b1e9f9f3499ce5bea5R313